exercises/csharp/TriviaGame/TriviaGame.Core/Game.cs

```diff
@@ -0,0 +1,124 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace TriviaGame.Core
+{
+  public class Game
+  {
+    public static readonly int NUMBER_OF_BOARD_SQUARES = 12;
+    public static readonly int MINIMUM_PLAYERS = 2;
+    public static readonly int MAX_QUESTIONS_PER_CATEGORY = 50;
+
+    public bool IsPlayable => NumberOfPlayers >= MINIMUM_PLAYERS;
+    public int NumberOfPlayers => _players.Count;
+
+    private readonly string[] _categories = { "Pop", "Science", "Sports", "Rock" };
+    private readonly Dictionary<string, List<string>> _questions = new Dictionary<string, List<string>>();
+
+    private readonly List<Player> _players = new List<Player>();
+    private Player _commander => _players[_currentPlayer];
```

Comment:
Is the name commander a C# convention? It's a little confusing as to what a commander is..